### PR TITLE
Extend window properties

### PR DESCRIPTION
**Pull request by shimil**

### CODE REVIEW

File: src/local.js [hunk 1]
```diff
@@ -170,6 +173,37 @@ function applyMessages(queueIndex, messages) {
...
+        devicePixelRatio: win.devicePixelRatio,
+        innerWidth: win.innerWidth,
+        innerHeight: win.innerHeight
+      }
+    }
+  }]);
+}
+
+function registerToWindowChanges(callback) {
+  win.addEventListener('orientationchange', callback);
```
Comment (liors) [inline src/local.js]: @shimil - what about win.addEventListener("resize")

Comment (shimil) [inline src/local.js]: @liors you're correct. Adding it now.

Comment (shimil) [inline src/local.js]: @liors I pushed it, please take a look.

Comment (liors) [inline src/local.js]: 👍

**Pull request merged**